assets/terraform-modules/packet/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -138,9 +137,21 @@ systemd:
     - name: "ip6tables-restore.service"
       enabled: true
       enable: true
-    - name: "iscsid.service"
-      enabled: true
+    - name: "create-iscsi-initiatorname-file.service"
       enable: true
+      contents: |
+        [Unit]
+        Description=Create ISCSI initiator name file
+        Before=iscsid.service
+        [Service]
+        Type=oneshot
+        RemainAfterExit=true
+        ExecStart=/opt/create-iscsi-initiatorname
+        [Install]
+        WantedBy=multi-user.target
+        RequiredBy=iscsid.service
+    - name: iscsid.service
```

Comment:
If we keep the quotes as they were we make diff more readable.

Reply:
Removed that commit.